Tests/govuk_ios/govuk_ios_unit_tests/Specs/Validators/StringValidatorTests.swift

```diff
@@ -0,0 +1,43 @@
+import Testing
+import Foundation
+
+@testable import govuk_ios
+
+@Suite
+struct StringValidatorTests {
```

Comment:
I think there is a missed opportunity here to actually check the behaviour of this class using a mock, rather than just checking the lengths.
Same thing is done in the redactors